cmd/kubelet/app/server_windows.go

```diff
@@ -37,7 +37,7 @@ func checkPermissions() error {
 
 	// For Windows user.UserName contains the login name and user.Name contains
 	// the user's display name - https://pkg.go.dev/os/user#User
-	klog.InfoS("Kubelet is running as", "login name", u.Username, "dispaly name", u.Name)
+	klog.InfoS("kubelet is running as", "login name", u.Username, "dispaly name", u.Name)
 
```

Comment:
Sorry for being unclear in my previous comment. All **info** log messages should be capitalized.